### ISSUE TITLE
Consider archiving this repository

### ISSUE THREAD

**Issue by chrisjsewell** [executablebooks/sphinx-tomyst#106]
Similar to https://github.com/executablebooks/rst2myst/issues/5,
I would consider archiving this repository, in favour of https://github.com/executablebooks/rst-to-myst

although I note this one has been updated more recently, so won't do it without discussion

cc @choldgraf @AakashGfude @mmcky @jstac for comment

**Comment by choldgraf** [comment]
I don't understand the comparative differences between this and `rst-to-myst`, so I'll defer to others there. But in general, if they have heavily overlapping functionality, I am +1 on deprecating one of them. One possibility is that if this repository is still needed by the QuantEcon team, we could just move it to the QE org?

**Comment by mmcky** [comment]
@chrisjsewell my plan is to move this to QuantEcon org once we are finished migrating all our major projects. 

- [x] https://python-programming.quantecon.org/intro.html
- [x] https://python.quantecon.org/intro.html 
- [ ] https://python-advanced.quantecon.org/intro.html (Due: Monday, 28th June)
- [ ] https://datascience.quantecon.org/intro.html (Due: Monday, 28th June)
- [ ] https://julia.quantecon.org/intro.html (Due: July)

`sphinx-tomyst` has been a very useful migration tool. 

@choldgraf this extension basically allowed you to add `sphinx_tomyst` as a sphinx extension and run `make myst` to generate a `myst` version of a `sphinx project`. But I haven't used `rst-to-myst` much to see how it migrates projects. I think direct translation of `rst-to-myst` is `technically` superior approach so happy to move `sphinx-tomyst` to the QuantEcon organisation. The downside of `sphinx-tomyst` is it mainly works for `core` rst (i.e. `docutils + sphinx`) as it infers from `sphinx.ast`

I was thinking of writing a migration guide for `sphinx rst` projects so I will certainly give `rst-to-myst` a spin. @chrisjsewell does this work well for migrating projects in bulk? Last time I gave it a spin it translated single files.

**Comment by chrisjsewell** [comment]
Yeh once https://github.com/executablebooks/rst-to-myst/pull/18 is merged (hopefully tonight) it will essentially be a lossless conversion and also output nice formatted Markdown with https://github.com/executablebooks/mdformat-myst.
This is something you just cannot do with the "standard" docutils AST, which this package works on, because there is literally no way to know for sure what the original text was e.g. what the original roles/directives used were.
Obviously its great that it worked for you but yeh, advertising it as a general tool, you would very rapidly rack up people opening bug issues

> does this work well for migrating projects in bulk? Last time I gave it a spin it translated single files.

still single file at present, but its pretty easy to add in-place conversions + file globbing, and so have a command that walks through and converts every file in a project, i.e. there is no requirement for any extra "inter-document" logic, you just point it at whatever text you want and it will convert it.
I'll probably also do this tonight/today as well.

**Comment by mmcky** [comment]
@chrisjsewell i agree. I'm happy not to maintain this as soon as rst-to-myst is good to go. Happy to also debug and test any bulk conversion / project conversion feature. Thanks @chrisjsewell
